testing/catalog.go

```diff
@@ -203,6 +268,19 @@ func (sw *SimpleWatch) Handle(m eirinix.Manager, e watch.Event) {
 	sw.Handled = append(sw.Handled, e)
 }
 
+// SimpleWatcher returns a dummy watcher
+func (c *Catalog) SimpleWatcherChannel(channel chan watch.Event) eirinix.Watcher {
```

Comment:
`SimpleWatcherWithChannel` or `SimpleChannelWatcher` ? to make it obvious that it's a Watcher